lib/spack/spack/cmd/module.py

```diff
@@ -257,6 +257,18 @@ def module(parser, args):
     }
     query_args = constraint_qualifiers.get(args.subparser_name, {})
     specs = args.specs(**query_args)
+    if args.subparser_name == 'loads':
+        if len(specs) > 1:
+            message = ("Trying to load multiple modules, please be more "
+                       "specific. Currently we would load: \n\t" +
+                       "\n\t".join(str(s) for s in specs) + "\n")
```

Comment:
An improvement here would be to use `spack.cmd.display_specs` to display specs, as it will be consistent with all the other commands. I don't think we say 'Aborting.' anywhere else, so I would also remove that.